skunkworks_crow/src/main/res/layout/fragment_forms.xml

```diff
@@ -11,43 +11,65 @@
         android:layout_centerHorizontal="true"
         android:layout_centerVertical="true"
         android:gravity="center"
-        android:textSize="22sp"
-        android:padding="10dp"/>
+        android:padding="10dp"
+        android:textSize="22sp" />
 
     <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recyclerview"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:id="@+id/recyclerview"
-        android:layout_above="@id/buttonholder"/>
+        android:layout_above="@id/buttonholder" />
+
 
     <LinearLayout
         android:id="@+id/buttonholder"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="60dp"
+        android:baselineAligned="false"
         android:layout_alignParentBottom="true"
-        android:orientation="horizontal"
-        android:weightSum="2">
+        android:background="@color/white"
+        android:orientation="horizontal">
 
-        <Button
-            android:id="@+id/toggle_button"
+        <LinearLayout
             android:layout_width="0dp"
             android:layout_height="match_parent"
             android:layout_weight="1"
-            android:padding="12dp"
-            android:text="@string/select_all"
-            android:textAllCaps="false"
-            android:textSize="16sp" />
+            android:gravity="center"
+            android:padding="10dp">
+
+            <Button
```

Comment:
`toggle_button` has so many attrs the same as `send_button` , consider extract them into `style.xml`.